docker/src/main/java/org/jclouds/docker/compute/options/DockerTemplateOptions.java

```diff
@@ -94,9 +94,10 @@
    protected Map<Integer, Integer> portBindings = ImmutableMap.of();
    protected String networkMode;
    protected Map<String, String> extraHosts = ImmutableMap.of();
+   protected Boolean privileged;
```

Comment:
Better make this a primitive boolean; there's no need to have to deal with nulls